code/game/objects/items/pinpointer.dm

```diff
@@ -158,6 +162,27 @@
 	if(!target) //target can be set to null from above code, or elsewhere
 		active = FALSE
 
+/obj/item/pinpointer/crew/prox //Weaker version of crew monitor primarily for EMT
+	name = "proximity crew pinpointer"
+	desc = "A handheld tracking device that displays its proximity to crew suit sensors."
+	icon_state = "pinpointer_crewprox"
+	custom_price = 40
+
+/obj/item/pinpointer/crew/prox/update_direction_icon(here, there)
+	var/size = ""
+	if(here == there)
```

Comment:
get_dist returns -1 if there are errors and here == there is considered an error.